app/models/miq_report/seeding.rb

```diff
@@ -1,76 +1,72 @@
 module MiqReport::Seeding
   extend ActiveSupport::Concern
 
-  module ClassMethods
-    REPORT_DIR  = File.expand_path(File.join(Rails.root, "product/reports"))
-    COMPARE_DIR = File.expand_path(File.join(Rails.root, "product/compare"))
+  REPORT_DIR  = Rails.root.join("product/reports")
+  COMPARE_DIR = Rails.root.join("product/compare")
 
+  module ClassMethods
     def seed
-      # Force creation of model instances for all report yaml files that exist in the product/reports directories
-      # that don't already have an instance in the model
-      MiqReport.sync_from_dir("report")
-      MiqReport.sync_from_dir("compare")
-    end
-
-    def seed_report(pattern, type = "report")
-      dir = type == "report" ? REPORT_DIR : COMPARE_DIR
-      files = Dir.glob(File.join(dir, "**/*#{pattern}*"))
-      files.collect do |f|
-        sync_from_file(f, dir, type)
+      reports = where(:rpt_type => 'Default').where.not(:filename => nil).index_by do |f|
```

Comment:
I don't understand the point of the `where.not(:filename => nil)`, but I'm assuming legacy reasons, so I'm leaving it for now, unless you think it can go.

Reply:
Maybe those are from miq expression editor or something?